src/applications/health-care-supply-reordering-redirect/sass/health-care-supply-reordering-redirect.scss

```diff
@@ -0,0 +1 @@
+// This file intentionally left blank.
```

Comment:
Note: The build in content-build is complaining it can't find `health-care-supply-reordering-redirect.css`. Unsure if this file is really needed to satisfy that, but this won't hurt either.